template/Makefile.in

```
@@ -199,7 +199,8 @@ DTRACE_DEPENDENT_OBJS = array.$(OBJEXT) \
 			parse.$(OBJEXT) \
 			string.$(OBJEXT) \
 			symbol.$(OBJEXT) \
-			vm.$(OBJEXT)
+			vm.$(OBJEXT) \
+			yjit.$(OBJEXT)
```

```suggestion
			vm.$(OBJEXT) \
			$(YJIT_OBJ)
```
YJIT requires `rustc` now and is an opt-in when configuring. Without the opt-in `yjit.o` is not built and we shouldn't need to rebuild it.

---

Ah, I see. I tested this on 3.1.2 that doesn't have Rust based YJIT yet. Thanks!